build.sbt

```diff
@@ -6,7 +6,7 @@ lazy val root = (project in file(".")).
   settings(commonSettings: _*).
   settings(
     sbtPlugin := true,
-    crossSbtVersions := Vector("0.13.18", "1.2.7"),
+    crossSbtVersions := Vector("0.13.18", "1.3.2"),
```

Comment:
This would bump the minimum required sbt version to 1.3.x, so I'd suggest we keep it at 1.2.x.

Reply:
Bumping this to 1.2.8 would be great.